website/cue/reference.cue

```diff
@@ -336,14 +336,14 @@ _values: {
 }
 
 #TargetTriples: {
-	"aarch64-unknown-linux-gnu":      bool
-	"aarch64-unknown-linux-musl":     bool
-	"armv7-unknown-linux-gnueabihf":  bool
-	"armv7-unknown-linux-musleabihf": bool
-	"x86_64-apple-darwin":            bool
-	"x86_64-pc-windows-msv":          bool
-	"x86_64-unknown-linux-gnu":       bool
-	"x86_64-unknown-linux-musl":      bool
+	"aarch64-unknown-linux-gnu":      *true | bool
```

Comment:
Not a big deal, but we've generally put the defaults at the end of unions like: `bool | *true`